### PR TITLE
AdvancedSettingModel unit test changes

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/shared/components/configuration/xero-connector/xero-connector.component.html [hunk 1]
```diff
@@ -84,7 +84,7 @@ <h5 *ngIf="!xeroConnectionInProgress && showDisconnectXero && isXeroConnected" (
         </div>
       </div>
     </div>
-    <app-configuration-step-footer-section [ctaText]="!xeroConnectionInProgress ? (isOnboarding ? ConfigurationCtaText.SAVE_AND_CONTINUE : ConfigurationCtaText.SAVE) : ConfigurationCtaText.SAVING" [isButtonDisabled]="isContinueDisabled" (save)="continueToNextStep()"></app-configuration-step-footer-section>
+    <app-configuration-step-footer-section [ctaText]="!xeroConnectionInProgress || xeroConnectorForm.value.xeroTenant === '' ? (isOnboarding ? ConfigurationCtaText.SAVE_AND_CONTINUE : ConfigurationCtaText.SAVE) : ConfigurationCtaText.SAVING" [isButtonDisabled]="isContinueDisabled" (save)="continueToNextStep()"></app-configuration-step-footer-section>
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/xero-connector/xero-connector.component.html:87]: why is this changed?

Comment (DhaaraniCIT) [reply src/app/shared/components/configuration/xero-connector/xero-connector.component.html:87]: this is because, when the connector dropdown is loading the, the save button is also showing loading in the button

**Review by ashwin1111 — APPROVED** [review]
